lib/game.rb

```diff
@@ -145,7 +157,7 @@ def computer_shot_results
 
   def computer_fire_shot
     @computer_shot = @human_board.cells.keys.sample
-      if @human_board.valid_coordinate?(@computer_shot)
+      if @human_board.valid_coordinate?(@computer_shot) && @human_board.cells[@computer_shot].fired_upon? == false
```

Comment:
Added a check against if the cell has already been fired upon

Reply:
This is great! It satisfies not firing the same spot twice.